types/carbone/index.d.ts

```diff
@@ -49,6 +49,7 @@ export interface Options {
     tempPath?: string | undefined;
     templatePath?: string | undefined;
     lang?: string | undefined;
+    timezone?: string | undefined;
```

Comment:
I bet it would be super useful to make this a string literal union of valid values, since I can never remember if it’s `"America/New York"` or `"America/New_York"`. But this is fine for now.

Reply:
I agree. As noted in [the documentation](https://carbone.io/documentation.html#formatd-patternout-patternin-), the string value should be one of the [TZ database name](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).

I can add this in another PR as enhancement.